api/idea-card/models/idea-card.settings.json

```diff
@@ -60,6 +60,25 @@
         "Advanced"
       ],
       "required": true
+    },
+    "comments": {
+      "via": "idea_card",
+      "collection": "comment"
+    },
+    "discord": {
```

Comment:
Is this the discord ID of the commenter?

Reply:
That would be the discord ID of the person submitting the idea.

Reply:
What if the commenter doesn't use discord?

Reply:
Maybe I made a mistake, the user commenting won't need a discord id. That would be just for submitting an idea. I hope that it makes sense.

Reply:
Oh I see this is the ideas collection.